core/alerts/alerts-core/src/main/java/com/thinkbiganalytics/alerts/api/core/AggregatingAlertProvider.java

```diff
@@ -449,7 +465,7 @@ private Alert alertChange(Alert alert, AlertResponder responder, AlertManager ma
         AlertManager mgr = this.managers.get(srcId.sourceId);
 
         if (mgr != null) {
-            return getAlert(srcId.alertId, mgr)
+            return getAlert(srcId.alertId, mgr,true)
                 .filter(alert -> alert.isActionable())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)